src/k8s/pkg/k8sd/app/hooks_bootstrap.go

```diff
@@ -235,6 +241,7 @@ func onBootstrapControlPlane(s *state.State, initConfig map[string]string) error
 	cfg.Certificates.APIServerKubeletClientCert = certificates.APIServerKubeletClientCert
 	cfg.Certificates.APIServerKubeletClientKey = certificates.APIServerKubeletClientKey
 	cfg.APIServer.ServiceAccountKey = certificates.ServiceAccountKey
+	cfg.Certificates.ExtraSANs = bootstrapConfig.ExtraSANs
```

Comment:
Extra SANs are potentially node-level configuration, do not add it to cluster config.

Reply:
(If there SANs that all nodes should have, then all nodes should request those when bootstrapping/joining)